defaults/main.yml

```diff
@@ -152,7 +152,9 @@ ssh_root_login_ips: None
 ## 5.3.16 Ensure SSH Idle Timeout Interval is configured
 ssh_client_alive_interval: 300
 ssh_client_alive_count_max: 3
-## 5.4.1.1 Ensure password expiration is 365 days or less
+## 5.4.1 Ensure password creation requirements are configured
+pass_min_len: 14
+## 5.5.1.2 Ensure password expiration is 365 days or less
```

Comment:
300 days, no?

Reply:
its the password minimun length, not the password expiration

Reply:
Although it is set to 300, i dont think we should change the text because its the task's name defined by the CIS

![Screenshot from 2024-04-10 11-20-02](https://github.com/alivx/CIS-Ubuntu-20.04-Ansible/assets/57574705/3386e525-4430-422e-b811-a52f7b896658)

Besides that, it says "365 or less"

Reply:
la var ahi dice 300...

Reply:
ok